getting_started/first_2d_game/01.project_setup.rst

```diff
@@ -40,7 +40,7 @@ Your project folder should look like this.
 
 This game is designed for portrait mode, so we need to adjust the size of the
 game window. Click on *Project -> Project Settings* to open the project settings
-window and in the left column, open the *Display -> Window* tab. There, set
+window and, in the left column, open the *Display -> Window* tab. There, set
```

Comment:
```suggestion
window, in the left column open the *Display -> Window* tab. There, set
```

Reply:
I like that, but it seems we would still want the "and" in there, i.e., "There, set window, and in the left column open the..." What do you think?

Reply:
I don't think it's grammatically necessary, but feel free to add it if you want since it wouldn't be grammatically wrong

Reply:
Oops! I misquoted the line. Now I see that this is your proposal:

"Click on _Project -> Project Settings_ to open the project settings window, in the left column open the _Display -> Window_ tab."

So, yes, now I understand and I agree that it's perfectly fine. Thank you for your patience! I will now attempt to include your change while maintaining a single-commit PR. :)